posthog/api/ingestion_warnings.py

```diff
@@ -11,6 +11,8 @@
 
 
 class IngestionWarningsViewSet(TeamAndOrgViewSetMixin, viewsets.ViewSet):
+    base_scope = "not_supported"
```

Comment:
This is one of the best things about the approach! It forces the developer to implement something. And if it isn't supported via the API then it should be explicitly set.

Reply:
Makes sense. Reading "not_supported" at first though, I wasn't sure what is it that's not supported, and where. Supposedly these endpoints _are_ supported as in-app endpoints, just not externally. So I think it would be clearer that this is a value with special behavior and intended for in-app-only endpoints if we used `"INTERNAL"` instead of `"not_supported"`

Reply:
Sure